server.js

```diff
@@ -0,0 +1,14 @@
+const express = require ('express')
+const app = express();
+const port = 3003;
+
+
+app.get('/',(req,res)=>{
+  res.send("Hello World!")
+})
+app.get('/ping',(req,res)=>{
+  res.send("pong")
+})
+app.listen(port,()=>{
+  console.log(`Server is started on ${port}`)
+})
\ No newline at end of file
```

Comment:
It's a good practice to add a newline at the end of your files. This can prevent unnecessary merge conflicts and other issues. [medium]